<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>576</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>876</width>
    <height>576</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>876</width>
    <height>576</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>0.900000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
	background:#222222;
}
QListWidget{
	background:#222222;
	border:2px solid #404040;
}
QPushButton{
	border:2px solid #A0A0A0;
	background:#404040;
	color:silver;
	border-radius:4px;
}
QPushButton:Pressed{
	border:2px solid #A0A0A0;
	background:#A0A0A0;
	color:white;
}
QLabel{
	color:silver;
	padding:2px;
}
</string>
   </property>
   <widget class="QListWidget" name="sidebar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>211</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="w_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background:#A0A0A0	
}</string>
    </property>
    <property name="text">
     <string>Weapon Detection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="park_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Parking Lot Counter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vehicle_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Vehicle Monitoring</string>
    </property>
   </widget>
   <widget class="QPushButton" name="luggage_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Abandoned Luggage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signout_button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Sign Out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="import_button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>500</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Import Video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_detection_button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>500</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Stop Detection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="view_ss_button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>500</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>View Screen-Shot</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Weapon Detection</string>
    </property>
   </widget>
   <widget class="QLabel" name="tm">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>550</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Smart City Surveilance System Final Year Project 2023</string>
    </property>
   </widget>
   <widget class="QLabel" name="menu_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>WELCOME</string>
    </property>
   </widget>
   <widget class="QLabel" name="g_window">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>631</width>
      <height>411</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>451</width>
      <height>251</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	border-radius:8px;
	border:3px solid #404040;
	background:#A0A0A0;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="detect_button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>500</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Detect</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
